turbo.json

```diff
@@ -3,11 +3,26 @@
   "baseBranch": "origin/master",
   "pipeline": {
     "build": {
-      "dependsOn": ["^build"],
-      "outputs": ["dist/**", "api/**", "services/**", "subscribers/**"]
+      "dependsOn": [
+        "^build"
+      ],
+      "outputs": [
+        "dist/**",
+        "api/**",
+        "helpers/**",
+        "loaders/**",
+        "migrations/**",
+        "models/**",
+        "repositories/**",
+        "services/**",
+        "subscribers/**",
+        "utils/**"
```

Comment:
Plugins will output their compiled code at their root. Given a new plugin added to our monorepo, if it outputs a directory that is not within this list, Turborepo will not output it. This could lead to multiple issues that can be hard to troubleshoot. This configuration should be revisited.